ui/cypress/plugins/index.js

```diff
@@ -36,37 +31,13 @@ const debug = (...args) => {
   );
 }
 
-/**
- *
- **/
-async function sendFileToEndpoint(filePath, endpointUrl, newFilename) {
```

Comment:
Replaced with runtime cypress browser command (not on cypress nodejs engine, which is what this code was doing).